src/media/js/main.js

```diff
@@ -1,59 +1,45 @@
 console.log('Sample Commonplace App');
 
-define(
-    'main',
-    [
-        'helpers',  // Must come before mostly everything else.
-        'helpers_local',
-        'forms',  // Comment this if your app has no forms.
-        'l10n',
-        'log',
-        'login',  // Comment this if your app does not have accounts.
-        'navigation',
-        'templates',
-        'user',  // Comment this if your app does not have accounts.
-        'z'
-    ],
-function() {
-    var console = require('log')('main');
-    var z = require('z');
-
-    console.log('Dependencies resolved, starting init');
-
-    z.body.addClass('html-' + require('l10n').getDirection());
+define('main', ['init'], function() {
```

Comment:
Fireplace depends on `settings_app` and `routes` here now. You might want to do that also.